### ISSUE TITLE
Set order that test suite builds / submits?

### ISSUE THREAD

**Issue by mnlevy1981** [ESMCI/cime#1331]
This is a very low priority issue that I'm bring over from the CESM-Development/cime issue page because it would be nice if this got addressed at some point.

> The aux_pop_MARBL test suite takes 40 minutes to build / submit 8 tests from a geyser node. Most of these are short, but we do have an ERI test that takes significantly longer than any other test to run (50 minutes instead of 5 or less)... so it would be nice to have that be the first test built and submitted. Currently it's the 5th test built, 20 minutes after the first job is submitted, and I don't see an easy way to change that. Perhaps a "priority" field in the xml file?

I'm not familiar with the parallelism in `create_test`; my impression is that the individual tests are built in parallel but processed serially (i.e. Test A is built in parallel and submitted, then Test B is built in parallel and submitted, ...). If the goal is to have all the tests processed in parallel, then this issue is not as important... though I'd argue being able to say "build / submit tests 1-4 first and 5-8 next" would still be useful.

**Comment by jgfouca** [comment]
@mnlevy1981 create_test has parallelism both at the test level and for the individual builds.

**Comment by jgfouca** [comment]
@mnlevy1981 this is a good point, the most expensive tests should be started first.

**Comment by jedwards4b** [comment]
@jgfouca I was about to say that this is a testlist issue and not really acme, but I just checked the list returned from test_utils and it is ordered correctly:
```
Testnames: ['ERI.T62_g16.CECO.yellowstone_intel.pop-ciso', 'ERS_Ld5_D.T62_g16.CECO.yellowstone_intel.pop-ciso', 'SMS_Ld20.T62_g16.CECO.yellowstone_intel.pop-ecosys_shr_stream', 'ERS.T62_g37.CECO.yellowstone_intel.pop-ecosys_restore_gx3v7', 'ERS.T62_g37.CECO.yellowstone_intel.pop-ciso_transient', 'SMS_D.T62_g37.CECO.yellowstone_intel.pop-ecosys', 'SMS_Ld2_D.T62_g37.CECO.yellowstone_intel.pop-ciso_daily_r4_tavg', 'SMS.T62_g16.GECO.yellowstone_intel.pop-ecosys_daily_r4_tavg', 'ERS.T62_g37.GECO.yellowstone_intel.pop-cice_ciso', 'ERS.T62_g37.GECO.yellowstone_intel.pop-cice_ecosys_dust_flux_driver', 'ERS.T62_g37.GECO.yellowstone_intel.pop-cice_ecosys_ladjust_bury_coeff', 'ERS.T62_g37.GECO.yellowstone_intel.pop-cice_ecosys_apply_nhx_surface_emis', 'ERS_Ld5_D.T62_g37.GECO.yellowstone_intel.pop-cice_ciso', 'SMS_D.T62_g37.GECO.yellowstone_intel.pop-cice_ecosys']

**Comment by jgfouca** [comment]
@jedwards4b so testlists already returns test in order of test time (longest to shortest)? Does that mean test_scheduler isn't honoring this order?

**Comment by jedwards4b** [comment]
Yes - I think that's what is happening.
